src/Server/Server.fsproj

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk.Web">
-    <PropertyGroup>
-        <OutputType>Exe</OutputType>
-        <TargetFramework>netcoreapp2.1</TargetFramework>
-    </PropertyGroup>
+  <PropertyGroup>
+      <OutputType>Exe</OutputType>
+      <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
So that means we're on net core 3 now. Have you tried running it on non-Windows machine? Last time I tried core3 it was a bit unstable with F#

Reply:
Well, a month ago it they made it [generally available](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0/) (before it was a preview), so I would expect it being stable. I did not try it myself though.